src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/RenderedViewModelBase.cs

```diff
@@ -0,0 +1,25 @@
+namespace NuGet.PackageManagement.UI.ViewModels
+{
+    public class RenderedViewModelBase : ViewModelBase
```

Comment:
I like the idea of the class name and its property describing the concept that this VM adds a title on top of the ViewModel base and it's a "page" that can be presented however we want: as a tab, as a section on a page, as a dialog, etc. etc. and this class becomes reusable.

```suggestion
    public class TitledPageViewModelBase : ViewModelBase
```